server/controllers/refer/courseParticipationsController.ts

```diff
@@ -48,6 +48,29 @@ export default class CourseParticipationsController {
     }
   }
 
+  details(): TypedRequestHandler<Request, Response> {
+    return async (req, res) => {
+      TypeUtils.assertHasUser(req)
+      const { courseParticipationId, referralId } = req.params
+
+      await this.courseService.getParticipation(req.user.token, courseParticipationId)
+
+      const referral = await this.referralService.getReferral(req.user.token, referralId)
+      const person = await this.personService.getPerson(req.user.username, referral.prisonNumber)
+
+      if (!person) {
+        throw createError(404, `Person with prison number ${referral.prisonNumber} not found.`)
+      }
+
+      res.render('referrals/courseParticipations/details', {
+        courseParticipationId,
+        pageHeading: 'Add Accredited Programme history',
```

Comment:
Maybe one for @Gweaton - have we got anywhere with revising these page headings? Would be good to have unique page titles for accessibility. This is the same as `/new` page